demo/web/src/main/notebooks/A. Real-time table ops/A4 Group and aggregate .md

```diff
@@ -0,0 +1,54 @@
+# Grouping and aggregating time series data
+
+In our previous notebook, we showed how to do time series and relational joins on our data. In this notebook, we show how to perform groupings and aggregations with our time series data.
+
+Let's start again by simulating some real-time data. This time we'll use a shorter time interval.
+
+```python
+time_interval = expressionToNanos("T1S")
```

Comment:
should you have the imports so this works encase they don't do the prior one?

Reply:
Yeah I thought about that and I think you're right. I'll change this (and the other notebooks) to be more verbose